### DISCUSSION TITLE
Audio waveform visualization

### DISCUSSION THREAD

**Discussion by andreidimaano** [remotion-dev/remotion#2375] [Q&A]
I really want to recreate this using remotion. Is it possible to create the waveforms using remotion? 

https://github.com/remotion-dev/remotion/assets/56818175/80c66b92-f159-49d0-9370-28c75f914bf4

**Comment by JonnyBurger** [comment]
Certainly!

Use `getWaveformPortion()` to get the bars of a specific portion of the audio: https://www.remotion.dev/docs/get-waveform-portion
You can achieve the effect of the waveform moving from the right to the left by playing around with the time window that you choose for visualization.
